src/helpers/reports.js

```diff
@@ -203,7 +203,7 @@ export const getSupportLevel = (grade, percentile, rawScore, taskId) => {
       tag_color: 'white',
     };
   }
-  if (percentile !== undefined && grade < 6) {
+  if (percentile !== undefined && (grade < 6 || grade === 'K')) {
```

Comment:
Can we use the [`getGrade` function](https://github.com/yeatmanlab/roar-utils/blob/94cad5ebfde63ecaabe21bc1fc4695c3274dbeca/src/utils.js#L232) from roar-utils to handle lots of different kinds of grade input?